packages/application/src/ErrorDialogContent.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { caretDownIcon, caretRightIcon } from '@jupyterlab/ui-components';
+
+import * as React from 'react';
+
+const MESSAGE_DISPLAY = 'elyra-pipelineSubmission-messageDisplay';
+const ERROR_DISPLAY_BUTTON = 'elyra-pipelineSubmission-errDisplayButton';
+const ERROR_DETAILS = 'elyra-pipelineSubmission-errDetails';
+const ERROR_DETAILS_VISIBLE = 'elyra-pipelineSubmission-error-visible';
+const ERROR_DETAILS_HIDDEN = 'elyra-pipelineSubmission-error-hidden';
```

Comment:
nice catch, that actually just poor naming since the function that utilizes it used to be called `pipelineSubmission`. I'll update the css names to be generic